strawberryfields/engine.py

```diff
@@ -280,9 +282,7 @@ def _broadcast_nones(val, dim):
 
             self._run_program(p, **kwargs)
             shots = kwargs.get("shots", 1)
-            self.samples = [
-                _broadcast_nones(p.reg_refs[k].val, shots) for k in sorted(p.reg_refs)
-            ]
+            self.samples = [_broadcast_nones(p.reg_refs[k].val, shots) for k in sorted(p.reg_refs)]
```

Comment:
How come the formatting changed here? Was this file not previously blacked?

Reply:
It seems like it :thinking: just ran `black -l 100` on it

Reply:
I'm guessing that only `black` (without `-l 100`) was called before, causing the line-break.